README.md

```diff
@@ -238,6 +238,7 @@ _See also: [WeblogMatrix](https://www.weblogmatrix.org/)_
 
 - [Antville](https://antville.org) - Free, open source project aimed at the development of a high performance, feature rich weblog hosting software. ([Source Code](https://github.com/antville/antville)) `Apache-2.0` `Javascript`
 - [Blog](https://github.com/m1k1o/blog) - Facebook-styled blog. Free, extremely lightweight, single-user and easy to install. `GPL-3.0` `PHP`
+- [btw](https://www.btw.so) - Open source Medium alternative. ([Source Code](https://github.com/btw-so/btw)) `GPL-3.0` `Nodejs`
```

Comment:
- Please change `Nodejs` to `Docker`. This seems to be the only option for hosting.
> Replace the variables in deploy/docker-compose.dev.yml

Reply:
Additionally, please change the description to `Personal website builder, alternative to Medium.` (the description should explain what the software does)